running-coreos/index.md

```diff
@@ -0,0 +1,35 @@
+Please follow the instruction below to know how to verify CoreOS images.
+
+First of all you have to install `gpg` tool then you have to download public image signing key:
+
+```sh
+curl -O https://coreos.com/security/image-signing-key/CoreOS_Image_Signing_Key.asc
+```
+
+Import public key and verify its ID with one on https://coreos.com/security/image-signing-key/ page:
+
+```sh
+gpg --import --keyid-format LONG CoreOS_Image_Signing_Key.asc
+gpg: key 50E0885593D2DCB4: public key "CoreOS Buildbot (Offical Builds) <buildbot@coreos.com>" imported
+gpg: Total number processed: 1
+gpg:               imported: 1  (RSA: 1)
+gpg: 3 marginal(s) needed, 1 complete(s) needed, PGP trust model
+gpg: depth: 0  valid:   2  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 2u
+```
+
+Download CoreOS image and GPG signature, i.e. QEMU production image:
+
+```sh
+curl -O http://stable.release.core-os.net/amd64-usr/current/coreos_production_qemu_image.img.bz2
+curl -O http://stable.release.core-os.net/amd64-usr/current/coreos_production_qemu_image.img.bz2.sig
+```
+
+Verify image with `gpg` tool:
+
+```sh
+gpg --verify coreos_production_qemu_image.img.bz2.sig
+gpg: Signature made Tue Jun 23 09:39:04 2015 CEST using RSA key ID E5676EFC
+gpg: Good signature from "CoreOS Buildbot (Offical Builds) <buildbot@coreos.com>"
+```
+
+The `Good signature` message indicates that the file signature is valid.
```

Comment:
The `Good signature` message indicates that the file signature is valid. Go launch some machines now that we've successfully verified that this CoreOS image isn't corrupt, that it was authored by CoreOS, and wasn't tampered with in transit.